config/urls.py

```diff
@@ -1,22 +1,15 @@
 from django.contrib import admin
 from django.urls import include, path
-from rest_framework.authentication import SessionAuthentication
-from rest_framework.documentation import include_docs_urls
-from rest_framework.permissions import IsAdminUser
+
+
+def trigger_error(request):
+    division_by_zero = 1 / 0  # noqa: F841 # flake8 : noqa
+
 
 urlpatterns = [
     path("", include("sidekick.urls"), name="sidekick"),
     path("admin/", admin.site.urls),
-    path("redcap/", include("rp_redcap.urls"), name="rp_redcap"),
-    path("asos/", include("rp_asos.urls"), name="rp_asos"),
     path("transferto/", include("rp_transferto.urls"), name="rp_transferto"),
-    path(
```

Comment:
I don't think we should remove the docs?

Reply:
This docs is the one that is causing this error

AssertionError: `coreapi` must be installed for schema support.

Reply:
Lets discuss in slack